modules/extensions/cilium.tf

```diff
@@ -52,19 +54,9 @@ locals {
 
     k8s = {
       requireIPv4PodCIDR   = true # wait for Kubernetes to provide the PodCIDR (ipam kubernetes)
```

Comment:
In the Cilium docs (https://docs.cilium.io/en/stable/network/concepts/ipam/kubernetes/), they mention the following:

"ipam: kubernetes: Enables Kubernetes IPAM mode. Enabling this option will automatically enable k8s-require-ipv4-pod-cidr if enable-ipv4 is true and k8s-require-ipv6-pod-cidr if enable-ipv6 is true."

Should we consider setting `enable-ipv4` to true?